.gitignore

```diff
@@ -1,3 +1,6 @@
+# Default database directory
+.unassigner/
```

Comment:
This should be in the user's home directory (`~/.unassigner`), not in the working directory.